tests/spdx/writer/tagvalue/test_tagvalue_writer_helper_functions.py

```diff
@@ -1,13 +1,6 @@
-# Copyright (c) 2023 spdx contributors
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#   http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+# SPDX-FileCopyrightText: 2023 spdx contributors
+
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Do you know if there is any convention about an empty line between the license header and the imports / actual content? I think that we are not consistent and I would prefer no empty line but this is probably also a matter of taste and not as important overall.

Reply:
I used `reuse` to add the license headers and it inserted a blank line. I think it is probably not important.

Reply:
Good to know!